tests/test_converter.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Graz University of Technology.
+#
+# invenio-records-lom is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Converter tests."""
+
+from invenio_rdm_pure.source.rdm.converter import Converter
+
+def test_conversion(pure_record_merged):
```

Comment:
what is this function for?

Reply:
When Pytest runs multiple test, the `test_` prefix makes them identifiable. 
https://docs.pytest.org/en/stable/getting-started.html#run-multiple-tests